src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/integration/PrisonApiMockServer.kt

```diff
@@ -660,4 +663,49 @@ class PrisonApiMockServer : WireMockServer(9005) {
         )
     )
   }
+
+  fun stubGetMovementSuccess(agencyId: String, fromDate: LocalDateTime, toDate: LocalDateTime) {
+    stubFor(
+      get("/api/movements/$agencyId/in?fromDateTime=$fromDate&toDateTime=$toDate")
+        .willReturn(
```

Comment:
meh.. could check for presence of header..

Reply:
done